tsconfig-test.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es6",
+		"strict": true,
+		"declaration": true,
+		"strictNullChecks": true,
+		"esModuleInterop": true,
+		"noImplicitReturns": true,
+		"noFallthroughCasesInSwitch": true,
+		"declarationMap": true,
+		"outDir": "./dist_test",
+	},
+	"include": [
+		"./test/**/*.ts",
+	],
+}
```

Comment:
Was this eventually supposed to be merged with the original?

Reply:
I kept it separate to avoid compiling the tests into the distributed package.